src/protagonist/Hydra/Model/OperationType.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Runtime.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Hydra.Model;
```

Comment:
This is a DLCS specific thing, rather than a Hydra concern (AFAIK anyway, unless they have this concept too!)